assets/js/googlesitekit/datastore/site/info.js

```diff
@@ -161,10 +162,23 @@ export const selectors = {
 	 * @param {Object} state Data store's state.
 	 * @return {(string|undefined)} This site's admin URL.
 	 */
-	getAdminURL: createRegistrySelector( ( select ) => () => {
+	getAdminURL: createRegistrySelector( ( select ) => ( state, page = undefined, args = {} ) => {
 		const { adminURL } = select( STORE_NAME ).getSiteInfo() || {};
 
-		return adminURL;
```

Comment:
Rather than the slightly complex logic in lines 172ff, I think it would be simpler to return early here if `adminURL` is `undefined`.

Reply:
Yeah, that makes sense. Now that I think about it, we can return early `adminURL` early if either `adminURL` or `page` are `undefined`, since we'll either be returning `undefined` or the URL without any additional args.